CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * [#68](https://github.com/nrepl/nREPL/issues/68): Avoid illegal access warning on JDK 9+ caused by `nrepl.middleware.interruptible-eval/set-line!`.
 * [#77](https://github.com/nrepl/nREPL/issues/77): Exit cleanly after pressing `ctrl-d` in an interactive REPL.
 * [#13](https://github.com/nrepl/nREPL/issues/13): Catch ThreadDeath exception thrown by interrupt.
+* [#61](https://github.com/nrepl/nrepl/issues/61): Implement a Transit transport
```

Comment:
This should be reflected in the manual, not in the changelog (as it's a different project).

Reply:
You mean update the docs at https://nrepl.xyz/nrepl/design/transports.html?

Reply:
Or the list of third-party extensions.

Reply:
Added fastlane to https://github.com/nrepl/nrepl/wiki/Extensions